core/templates/domain/question/QuestionObjectFactory.ts

```diff
@@ -192,9 +209,15 @@ export class QuestionObjectFactory {
   constructor(
     private stateObject: StateObjectFactory) {}
 
+  /* Null in id denotes the a new question whose id is yet to be
+  set, this id is layer set in backend API service. Reagrding statename
+  as question, this statename is used throughout creating the state
+  for question like in createFromBackendDict. This function cannot be
+  removed as it is used to create a temperory default question in question
+  editor until the question is once saved. */
   createDefaultQuestion(skillIds: string[]): Question {
     return new Question(
-      null, this.stateObject.createDefaultState(null),
+      null, this.stateObject.createDefaultState('question'),
```

Comment:
ditto with the `createDefaultState`, it should be called with `null` or not called at all.

Reply:
Made this change